### PR TITLE
chore: Use codec in package util

### PR DESCRIPTION
**Pull request by DongLieu**
I think we should use the util package in e2e instead of testutil.

### CODE REVIEW

**Review by pysel — COMMENTED** [review]
hey, thank you for the PR!

File: tests/e2e/initialization/node.go [hunk 1]
```diff
@@ -22,7 +22,6 @@ import (
 	"github.com/cosmos/cosmos-sdk/server"
 	srvconfig "github.com/cosmos/cosmos-sdk/server/config"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/types/module/testutil"
```
[the remark this answers is not in the document]
Comment (mattverse) [reply tests/e2e/initialization/node.go:25]: Its from the removal from 

```
	cfg := testutil.MakeTestEncodingConfig()
```

**Review by mattverse — APPROVED** [review]
Good catch. LGTM

**Comment by mattverse** [comment]
This PR includes modifications to the `tests/e2e/initialization` module.
            Please follow the instructions below:

            1. Backport these changes to the previous Osmosis version's branch.
            2. Run the script inside a Docker container to update genesis and configs for pre-upgrade Osmosis.
            3. Merge the backported changes.
            4. The image will be built and uploaded to Docker Hub [here](https://hub.docker.com/r/osmolabs/osmosis-e2e-init-chain/tags).
            5. Grab the latest image and update it in the PR to the main branch replacing the `previousVersionInitTag` in the `osmosis/tests/e2e/containers/config.go`

File: tests/e2e/initialization/node.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (pysel) [reply tests/e2e/initialization/node.go:25]: oh, i missed it, thanks!

**Comment by DongLieu** [comment]
> This PR includes modifications to the `tests/e2e/initialization` module. Please follow the instructions below:
> 
> ```
>         1. Backport these changes to the previous Osmosis version's branch.
>         2. Run the script inside a Docker container to update genesis and configs for pre-upgrade Osmosis.
>         3. Merge the backported changes.
>         4. The image will be built and uploaded to Docker Hub [here](https://hub.docker.com/r/osmolabs/osmosis-e2e-init-chain/tags).
>         5. Grab the latest image and update it in the PR to the main branch replacing the `previousVersionInitTag` in the `osmosis/tests/e2e/containers/config.go`
> ```

Do you mean require @osmo-bot to do it?

**Comment by mattverse** [comment]
No, I was wondering if you could add this as well! 🙏 
We need to update our docker image as well when we have any updates on e2e code base

**Comment by DongLieu** [comment]
I've never done that before. Can you be more specific about step number 2?

**Review by czarcas7ic — APPROVED** [review]

**Comment by czarcas7ic** [comment]
Since this is a minor change, its fine that this will eventually make its way into an init image, and the init image does not need to be generated now.

**Pull request merged**